lib/Db/AttachmentMapper.php

```diff
@@ -52,54 +51,43 @@ public function __construct(IDBConnection $db, CardMapper $cardMapper, IUserMana
 	}
 
 	/**
-	 * @param $id
-	 * @return Entity|Attachment
-	 * @throws \OCP\AppFramework\Db\DoesNotExistException
-	 * @throws \OCP\AppFramework\Db\MultipleObjectsReturnedException
+	 * @param int $id
+	 * @return Attachment
+	 * @throws DoesNotExistException
+	 * @throws MultipleObjectsReturnedException
+	 * @throws \OCP\DB\Exception
 	 */
 	public function find($id) {
 		$qb = $this->db->getQueryBuilder();
 		$qb->select('*')
 			->from('deck_attachment')
```

Comment:
I think it'd be better to use here: `->from($this->getTableName())` instead of the name of the table.

Reply:
Nice suggestion, added! (also for the rest of finders in this class)